project/AppDependencies.scala

```diff
@@ -9,27 +9,27 @@ import sbt._
 
 object AppDependencies {
 
-  val bootstrapPlayVersion = "8.1.0"
+  val bootstrapPlayVersion = "9.13.0"
 
   val compile: Seq[ModuleID] = List(
     ws,
     "uk.gov.hmrc"                  %% "bootstrap-backend-play-30" % bootstrapPlayVersion,
-    "org.typelevel"                %% "cats-core"                 % "2.9.0",
+    "org.typelevel"                %% "cats-core"                 % "2.13.0",
     "com.chuusai"                  %% "shapeless"                 % "2.4.0-M1",
     "com.neovisionaries"            % "nv-i18n"                   % "1.29",
-    "com.fasterxml.jackson.module" %% "jackson-module-scala"      % "2.15.2",
-    "com.github.jknack"             % "handlebars"                % "4.3.1"
+    "com.fasterxml.jackson.module" %% "jackson-module-scala"      % "2.19.1",
+    "com.github.jknack"             % "handlebars"                % "4.4.0"
   )
 
   def test(scope: String = "test, it"): Seq[sbt.ModuleID] = List(
-    "org.scalatest"       %% "scalatest"              % "3.2.15"             % scope,
+    "org.scalatest"       %% "scalatest"              % "3.2.19"             % scope,
```

Comment:
scalatest can be removed as it's already provided by bootstrap-test-play

Reply:
sorry yeah I had done this just not pushed up